tests/models/test_pyro.py

```diff
@@ -62,13 +99,9 @@ def test_pyro_bayesian_regression(save_path):
     trainer.fit(plan, train_dl)
 
     # test save and load
-    post_dl = AnnDataLoader(adata, shuffle=False, batch_size=128)
-    mean1 = []
-    with torch.no_grad():
-        for tensors in post_dl:
-            args, kwargs = model._get_fn_args_from_batch(tensors)
-            mean1.append(model(*args, **kwargs).cpu().numpy())
-    mean1 = np.concatenate(mean1)
+    quants = model.guide.quantiles([0.5])
+    sigma_median = quants["sigma"][0].cpu().detach().numpy()
```

Comment:
nit: it's a little cheaper to detach as early as possible: `.detach().cpu().numpy()`